tobuild.json

```diff
@@ -27,11 +27,10 @@
   "configuration-modules-core": {
     "branch": "master",
     "prs": [
-      1438,
-      1452,
-      1510
+      1695,
+      1697
```

Comment:
this one has been merged into master -> can be removed

Reply:
done